### ISSUE TITLE
[FEATURE] Look for updates

### ISSUE THREAD

**Issue by phulstaert** [Leantime/leantime#907]
It would be nice if we could periodically check for updates and give a notification to the admins if this is the case (by using Cron?).
People don't always check if new versions are available, since most software does this for them.

**Comment by marcelfolaron** [comment]
Agreed, I think we should combine that with an auto update script (download
and extract in a tmp folder and move files).

**Comment by phulstaert** [comment]
Does the cron service run with php-cli ? This often has no timeout and runs even when the site has no hits.

We should also create a variable from where we know it is a docker instance and encourage updating though docker (pulling the latest image) instead of doing an update. The because otherwise users won't receive newer php, mysql, apache versions and other security updates.
